chatgpt-plugin/src/pages/api/handlers/solana-pay/sign/createTransferAsset.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  SPL_ACCOUNT_COMPRESSION_PROGRAM_ID,
+  SPL_NOOP_PROGRAM_ID,
+} from "@solana/spl-account-compression";
+import { PublicKey, Transaction } from "@solana/web3.js";
+import { NextApiRequest } from "next";
+
+import { bs58 } from "@coral-xyz/anchor/dist/cjs/utils/bytes";
+import { createTransferInstruction } from "@metaplex-foundation/mpl-bubblegum";
+import { makeRespondToSolanaPayGet, makeRespondToSolanaPayPost } from ".";
+import configConstants, { CONNECTION } from "../../../constants";
+import { bufferToArray, getAsset, getAssetProof, getBubblegumAuthorityPDA } from "../utils/helpers";
+configConstants();
+
+async function createTransferAsset(req: NextApiRequest) {
+  const { destination, assetId } = req.query;
+  const { account: sender } = req.body;
+  if (!sender || !destination || !assetId) {
+    throw new Error("Missing required parameters");
+  }
+
+  const assetProof = await getAssetProof(assetId, CONNECTION.rpcEndpoint);
+  if (!assetProof?.proof || assetProof.proof.length === 0) {
+    throw new Error("Proof retrieved for the given assetId is empty. Please check the assetId.");
+  }
+  const proofPath = assetProof.proof.map((node: string) => ({
+    pubkey: new PublicKey(node),
+    isSigner: false,
+    isWritable: false,
+  }));
+
+  const rpcAsset = await getAsset(assetId, CONNECTION.rpcEndpoint);
```

Comment:
please use https://github.com/helius-labs/helius-sdk

Reply:
same reason as above for `getAssetProof`